taskcluster/ac_taskgraph/target_tasks.py

```diff
@@ -15,6 +15,14 @@ def filter(task, parameters):
     return [l for l, t in full_task_graph.tasks.iteritems() if filter(t, parameters)]
 
 
+@_target_task("nightly_release")
```

Comment:
Same comment about `nightly-release`

Reply:
I've renamed it, indeed it looks cleaner 👍